packages/forms/resources/views/components/tabs.blade.php

```diff
@@ -64,7 +64,7 @@
                 x-bind:tabindex="tab === '{{ $tab->getId() }}' ? 0 : -1"
                 class="filament-forms-tabs-component-button flex items-center gap-2 shrink-0 p-3 text-sm font-medium"
                 x-bind:class="{
-                    'text-gray-500 @if (config('forms.dark_mode')) dark:text-gray-400 @endif': tab !== '{{ $tab->getId() }}',
+                    'text-gray-500 hover:text-gray-800 focus:text-primary-600 @if (config('forms.dark_mode')) dark:text-gray-400 dark:hover:text-gray-200 dark:focus:text-primary-600 @endif': tab !== '{{ $tab->getId() }}',
```

Comment:
Is this separate class for dark mode still needed now?

```suggestion
                    'text-gray-500 hover:text-gray-800 focus:text-primary-600 @if (config('forms.dark_mode')) dark:text-gray-400 dark:hover:text-gray-200 @endif': tab !== '{{ $tab->getId() }}',
```

Reply:
Yeah I thought that too but found that dark text overwrites the focus
<img width="452" alt="Screenshot 2023-05-07 at 08 55 10" src="https://user-images.githubusercontent.com/533658/236665124-fd07ddf5-dbff-4f06-92fa-df955b588b63.png">

Reply:
Ok, wasn't too sure if this worked, but thanks for testing. 😊